pages/exercises/[lessonSlug].tsx

```diff
@@ -25,12 +26,45 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
 
   return (
     <Layout title={currentLesson.title}>
+      <ExercisesNavCard lessonDocUrl={currentLesson.docUrl} lessonSlug={slug} />
       <h1>{currentLesson.title}</h1>
       {alerts && <AlertsDisplay alerts={alerts} />}
     </Layout>
   )
 }
 
+type ExercisesNavCardProps = {
+  lessonDocUrl: string | null | undefined
```

Comment:
The reason why the lessonDocUrl property has type `string | null | undefined` is because the `Lesson` type has its docUrl property as `docUrl?: string | null`. I made lessonDocUrl `string | null | undefined` instead of `lessonDocUrl?: string | null` so that people calling `ExerciseNavCard` are forced to pass in a `lessonDocUrl` prop.